### ISSUE TITLE
Docker image build error while Graal install

### ISSUE THREAD

**Issue by florentk** [kiwix/metrics#59]
```
Processing /dist/graal-0.2.7-py3-none-any.whl
Collecting MarkupSafe>=0.23
Downloading MarkupSafe-1.1.1-cp35-cp35m-manylinux1_x86_64.whl (27 kB)
Collecting numpy>=1.13.3
Downloading numpy-1.19.0rc1.zip (7.3 MB)
Installing build dependencies: started
Installing build dependencies: finished with status 'done'
Getting requirements to build wheel: started
Getting requirements to build wheel: finished with status 'error'
ERROR: Command errored out with exit status 1:
command: /usr/bin/python3 /usr/local/lib/python3.5/dist-packages/pip/_vendor/pep517/_in_process.py get_requires_for_build_wheel /tmp/tmpk9bv285g
cwd: /tmp/pip-install-w57px_9q/numpy
Complete output (19 lines):
Traceback (most recent call last):
File "/usr/local/lib/python3.5/dist-packages/pip/_vendor/pep517/_in_process.py", line 280, in <module>
main()
File "/usr/local/lib/python3.5/dist-packages/pip/_vendor/pep517/_in_process.py", line 263, in main
json_out['return_val'] = hook(**hook_input['kwargs'])
File "/usr/local/lib/python3.5/dist-packages/pip/_vendor/pep517/_in_process.py", line 114, in get_requires_for_build_wheel
return hook(config_settings)
File "/usr/local/lib/python3.5/dist-packages/setuptools/build_meta.py", line 148, in get_requires_for_build_wheel
config_settings, requirements=['wheel'])
File "/usr/local/lib/python3.5/dist-packages/setuptools/build_meta.py", line 128, in _get_build_requires
self.run_setup()
File "/usr/local/lib/python3.5/dist-packages/setuptools/build_meta.py", line 250, in run_setup
self).run_setup(setup_script=setup_script)
File "/usr/local/lib/python3.5/dist-packages/setuptools/build_meta.py", line 143, in run_setup
exec(compile(code, __file__, 'exec'), locals())
File "setup.py", line 182
raise ValueError(f'Submodule {p} missing')
^
SyntaxError: invalid syntax
----------------------------------------
ERROR: Command errored out with exit status 1: /usr/bin/python3 /usr/local/lib/python3.5/dist-packages/pip/_vendor/pep517/_in_process.py get_requires_for_build_wheel /tmp/tmpk9bv285g Check the logs for full command output.
Removing intermediate container 4afebaa27aa3
The command '/bin/sh -c ${DIST_SCRIPT} --build --install --install_system --distdir ${DIST_DIR} --relfile ${REL_FILE}' returned a non-zero code: 1
```

**Comment by florentk** [comment]
seem related with https://github.com/numpy/numpy/issues/16294

**Comment by florentk** [comment]
The realease candidate of NumPy need Python 3.6, but the docker image use Python 3.5. To fix the problem, we use the previous version (release version) of NumPy.
fix with 40184b385726dc7a3e9e98df6dbf10772b710a92
